### ISSUE TITLE
Deploy breaks in 1.12 b/c docker.socket no longer part of distro

### ISSUE THREAD

**Issue by gdw2** [mongrelion/ansible-role-docker#1]
See: https://github.com/docker/docker/issues/25098#issuecomment-236512644

Here is the error I get:

```
RUNNING HANDLER [mongrelion.docker : restart docker] ***************************
fatal: [10.17.232.10]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to start docker.service: Unit docker.socket failed to load: No such file or directory.\n"}

```

If I try to start docker from the commandline:

```
[root@greg-deploy-test ~]# systemctl start docker
Failed to start docker.service: Unit docker.socket failed to load: No such file or directory.
```

If I uninstall and reinstall docker-engine (thus not copying over the modified `docker.service`), docker starts just fine.

Is there a reason you override the `docker.service` that ships with docker-engine?

**Comment by gdw2** [comment]
@brucellino, who forked the repo, noticed the problem too:

https://github.com/AAROC/ansible-role-docker/commit/1d8c4ee769ee5e7c03e1e762f453a3be798f9021

**Comment by mongrelion** [comment]
@gdw2 thanks for spotting this. I also ran into this issue but I totally missed this issue. The fix is on commit 8e1c463e6e5f24ef7821422f8c92e6020ab44f8f

Could you please re-check with the latest release of the role in the ansible galaxy and confirm that the issue is gone?

Cheers

**Comment by gdw2** [comment]
fixed. thanks.
